apps/frontend/core-dapp/src/stores/DepositTradeHelperStore.ts

```diff
@@ -0,0 +1,28 @@
+import { BigNumber } from 'ethers'
+import { ContractStore } from 'prepo-stores'
+import { RootStore } from './RootStore'
+import { SupportedContracts } from '../lib/contract.types'
+import { DepositTradeHelperAbi, DepositTradeHelperAbi__factory } from '../../generated/typechain'
+
+type WrapAndDeposit = DepositTradeHelperAbi['functions']['wrapAndDeposit']
+
+export class DepositTradeHelperStore extends ContractStore<RootStore, SupportedContracts> {
+  constructor(root: RootStore) {
+    super(root, 'DEPOSIT_TRADE_HELPER', DepositTradeHelperAbi__factory)
+  }
+
+  async wrapAndDeposit(
+    recipient: string,
+    value: BigNumber
+  ): Promise<{ success: boolean; error?: string }> {
```

Comment:
```suggestion
  ): Promise<{ success: true; } | { success: false; error: string }> {
```

I'm a bit picky with these kind of things 😁 but feel free to disregard.